libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/DropInNavigationView.kt

```diff
@@ -158,17 +157,19 @@ class DropInNavigationView @JvmOverloads constructor(
     private fun initMapLongClickListener() {
         binding.mapView.gestures.addOnMapLongClickListener { clickPoint ->
             ifNonNull(MapboxNavigationApp.current()) { mapboxNavigation ->
```

Comment:
Could move this inside the behavior. But it requires location permissions

Reply:
It was moved in this PR: https://github.com/mapbox/mapbox-navigation-android/pull/5496